### ISSUE TITLE
Feedback (during & after)

### ISSUE THREAD

**Issue by leahbannon2** [department-of-veterans-affairs/va.gov-team#76945]
Goal: Continuously improve during and after the offsites.

Links: 
- [List of all feedback I could find from previous offsites](https://dvagov.sharepoint.com/:x:/r/sites/OCTO/Shared%20Documents/General/Facilitation%20and%20Offsites/Feedback.xlsx?d=w1ed664102b284b0d99244fde30fd0cf8&csf=1&web=1&e=OdjAma)

Acceptance criteria: 
- [ ] Posters spread throughout the room with feedback prompts -- Need to decide on prompts.
- [ ] End-of-day delta and pluses exercise ("I need 3 in each category before we can leave.") -- Need someone to lead and someone to take detailed notes.
- [ ] Survey sent to attendees after the end of the offsite.
- [ ] Document feedback for future planning.

**Comment by leahbannon2** [comment]
Send invites early (to do 2nd?)
- weekly planning meetings
- feedback sessions with volunteers
- daily stand up/planning meetings in final weeks
- cal invites for attendees
-

**Comment by leahbannon2** [comment]
Add seating assignments as separate task from content?

**Comment by leahbannon2** [comment]
Standardize the [final, pre-travel reminders like this](https://dsva.slack.com/archives/C05295V44HE/p1712339736037219)
- what to pack
- what to wear
- travel stuff like duluth and we desperately need a better system than the last minute $1 credit limit surprise

**Comment by leahbannon2** [comment]
Maybe look for public libraries for offsite locations?

**Comment by leahbannon2** [comment]
I keep thinking I want to do an unconference wall like this next time. 

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/37309246/b624708a-6552-41cf-8e12-2c0a990fbdd8)
